core/src/main/scala/chisel3/Aggregate.scala

```diff
@@ -951,6 +951,11 @@ abstract class Record(private[chisel3] implicit val compileOptions: CompileOptio
             " See the cookbook entry 'How do I deal with the \"unable to clone\" error?' for more details."
         )
       }
+      if (field.binding.isDefined) {
+        throw RebindingException(
+          s"Cannot create Record ${this.className}; element ${field} of record is already a bound hardware."
```

Comment:
```suggestion
          s"Cannot create Record ${this.className}; element ${field} of record must be a Chisel type, not hardware."
```

to be consistent with that message. maybe we should pull that "must be a Chisel type, not hardware" into some common string so we're consistent with these sorts of errors?